tests/virtualization/xen/ssh_hypervisor.pm

```diff
@@ -24,16 +24,22 @@ use utils;
 
 sub run {
     my ($self) = @_;
-    select_console 'x11';
+    opensusebasetest::select_serial_terminal();
```

Comment:
You started to use a lot `select_serial_terminal()`, does it make sense with `use base "x11test";` ?

Reply:
It actually is:
```
use qam 'ssh_add_test_repositories';
use utils 'ssh_fully_patch_system';
```